include/mbgl/map/mode.hpp

```diff
@@ -10,6 +10,11 @@ enum class MapMode : uint8_t {
     Still, // a once-off still image
 };
 
-}
+enum class GLContextMode : uint8_t {
```

Comment:
Can you add a comment here about what the difference in behavior is between the two modes.